src/OmniSharp.Abstractions/Models/v1/FileMemberElement.cs

```diff
@@ -10,6 +10,8 @@ public class FileMemberElement : IComparable<FileMemberElement>
         public QuickFix Location { get; set; }
 
         public string Kind { get; set; }
+        
+        public ICollection<string> Features { get; } = new List<string>();
```

Comment:
It doesn't have to be an enum, we could simply make it a string instead, and that would work fine as well.